about.html

```diff
@@ -77,7 +77,7 @@ <h3>2013</h3>
   <footer id="copyright" class="cp-dark">
     <div class="wrapper">
       <div class="footer-logo">
-        <img src="./images/creative-commons.png" alt="Creative Commons">
+        <img src="./images/cc-logo-white.png" alt="Creative Commons">
       </div>
       <div class="cp-info">
         <p><a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a></p>
```

Comment:
- [ ] You'll notice that even though the desktop version of the about page has a dark footer, the mobile version instead has a dark partner's section and a white footer. Kindly address this issue so as to follow the design requirements 👍  
- Your design:

![Annotation 2022-05-19 040746](https://user-images.githubusercontent.com/85113567/169198252-b13d3d10-d545-4691-9631-1679fcbb831a.jpg)
- Required design:
![Annotation 2022-05-19 040803](https://user-images.githubusercontent.com/85113567/169198316-d32d93ef-3102-46bc-aa09-13435371cba5.jpg)